roles/mirror_ocp_release/tasks/facts.yml

```diff
@@ -122,6 +122,8 @@
   ansible.builtin.set_fact:
     bootstraposimage: "{{ mor_webserver_url }}/{{ ocp_release_data['rhcos_images']['qemu_location'] | basename }}?sha256={{ ocp_release_data['rhcos_images']['qemu_uncompressed_sha256'] }}"
     clusterosimage: "{{ mor_webserver_url }}/{{ ocp_release_data['rhcos_images']['openstack_location'] | basename }}?sha256={{ ocp_release_data['rhcos_images']['openstack_sha256'] }}"
+    metalosimage: "{{ mor_webserver_url }}/{{ ocp_release_data['rhcos_images']['metal_iso_location'] | basename }}?sha256={{ ocp_release_data['rhcos_images']['metal_iso_
```

Comment:
is this var used somewhere afterwards?

Reply:
yeah, That will be during ACM hub configurations to set the agent service config. So just covering a requirement and advance. Thanks!

Reply:
it looks like a new line character got in the way of the metal_iso_sha256 object name. I'm guessing with this change the job will succeed.

I'm looking forward to have this change verified.